pipeline/dbt/models/intermediate/odspep/int_odspep__duplicate_structures.sql

```diff
@@ -0,0 +1,30 @@
+WITH ressources_partenariales AS (
+    SELECT * FROM {{ ref('int_odspep__enhanced_res_partenariales') }}
+),
+
+unique_id_res AS (
+    SELECT DISTINCT ON (id_res)
+        id_res,
+        nom_structure,
+        l1_identification_dest_adr,
+        l3_complement_adr,
+        l4_numero_lib_voie_adr,
+        code_commune_adr
+    FROM ressources_partenariales
+),
+
+select_duplicates AS (
+    SELECT *,
+        '1'::BOOLEAN                                                                                                                                                     AS "duplicate",
+        COUNT(id_res) OVER(PARTITION BY UNACCENT(LOWER(CONCAT(nom_structure, l1_identification_dest_adr, l3_complement_adr, l4_numero_lib_voie_adr, code_commune_adr)))) AS "count_duplicates"
+ 
```

Comment:
On peut ajouter un identifiant de groupe:

```suggestion
RANK() OVER(ORDER BY group_key) AS "group_id",
```

Ca nous permet de clairement voir les regroupements et de l'utiliser par la suite !